src/components/dialogs/parameters/load-flow-parameters.js

```diff
@@ -605,6 +471,49 @@ export const LoadFlowParameters = ({ hideParameters, parametersBackend }) => {
         },
     ];
 
+    const [specificCurrentParams, setSpecificCurrentParams] = useState(
+        params['specificParametersPerProvider']
+    );
+
+    useEffect(() => {
```

Comment:
the fusion and the updateParameters should be done in the onSpecificParamChange. You don't need this useEffect I think.